Makefile

```diff
@@ -276,6 +276,11 @@ endif
 # This can be overridden by the platform.
 include lib/cpus/cpu-ops.mk
 
+ifeq (${COREBOOT},1)
+# Include coreboot support library when building with coreboot as BL2
+include lib/coreboot/coreboot.mk
+endif
+
```

Comment:
It shouldn't be necessary to change the generic Makefile. Platforms can just pull coreboot.mk as required.